internal/startup/auto_operator_configuration_handler.go

```diff
@@ -118,60 +144,6 @@ func (r *AutoOperatorConfigurationResourceHandler) validateOperatorConfiguration
 	return nil
 }
 
-func (r *AutoOperatorConfigurationResourceHandler) waitForWebserviceEndpoint(
-	ctx context.Context,
-	logger *logr.Logger,
-) error {
```

Comment:
The new readiness check shares most of the logic for waiting for the web service endpoint, so this has been moved to ready_check.go.